### ISSUE TITLE
Update to work with Ubuntu 14 by changing default variables

### ISSUE THREAD

**Issue by EmilStenstrom** [nicolai86/ansible-phantomjs#4]
Hi,
This module works great with latest Ubunto by just setting:

``` yaml
  vars:
    phantomjs_version: "1.9.8"
    phantomjs_url: "https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.8-linux-x86_64.tar.bz2"
```

Maybe it would be a good idea to change those, and make your module as compatible with "trusty" on ansible galaxy?

**Comment by nicolai86** [comment]
Hey,

any reason why you are not using phantomjs 2.0 yet? 
I'll update the URLs to target 2.0.

**Comment by EmilStenstrom** [comment]
Thanks!

**Comment by nicolai86** [comment]
thanks for keeping an eye on the phantomjs releases ;)

I've released 0.2.1 on ansible galaxy as well.
